src/pip/__main__.py

```diff
@@ -18,7 +18,7 @@
     path = os.path.dirname(os.path.dirname(__file__))
     sys.path.insert(0, path)
 
-from pip._internal.cli.main import main as _main  # isort:skip # noqa
+from pip._internal.cli.main import main as _main
```

Comment:
Hmm, I’m surprised the isort comment is not needed.

Reply:
This changed with isort 5, which now treats imports in blocks rather than just the top of the file. This is why it also lints our imports within the if TYPE_CHECKING blocks now.